pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -2447,8 +2447,8 @@ func TestEval(t *testing.T) {
 				}
 			}
 			`,
-			expectedStatusCode: http.StatusBadRequest,
-			expectedMessage:    "invalid condition: invalid query A: data source not found: unknown",
+			expectedStatusCode: http.StatusUnauthorized,
+			expectedMessage:    "user is not authorized to query one or many data sources used by the rule",
```

Comment:
Note: here (and in a few other cases) the returned error message differs, we should check with the alerting team if it's ok.